### DISCUSSION TITLE
Not able to build "feature/ui-automation" branch locally.

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#6167] [Q&A] [answered]
Hi,

I cloned the feature/ui-automation branch to test the `Automation` stuff and was trying to build it but was getting an error as `The name "WpfAvaloniaHost" does not exist in the namespace "clr-namespace:Avalonia.Win32.Interop.Wpf;assembly=Avalonia.Win32.Interop"`. I have a visual studio 2019 installed and the version is `16.10.2` (if that matters). 

After I removed the` WindowsInteropTest` project from Visual Studio (which was throwing this error), I am getting the errors shown in the screenshot below. I am clueless about what needs to be done to fix it (I have referred to the solution [here ](https://github.com/AvaloniaUI/Avalonia/issues/4257) and also tried cleaning and rebuilding the solution but no help). Any help would be much appreciated.

FYI: I am able to build the `master` branch and run the sample project `ControlCatalog.Desktop`

![image](https://user-images.githubusercontent.com/79670853/124193875-734a9680-da7c-11eb-962f-bb678850f1bb.png)

**Comment by maxkatz6** [comment, marked as answer]
You need to disable api compat checks
See https://github.com/AvaloniaUI/Avalonia/pull/4348 and https://stu.dev/check-for-breaking-changes-with-apicompat/

**Reply by maxkatz6** [reply]
This ui-automation has breaking changes, so that's why it throws these errors

**Reply by ShrutiJaiswal1494** [reply]
Thanks. It resolved all my ApiCompat issues.
